app/grandchallenge/verifications/models.py

```diff
@@ -131,6 +139,35 @@ def send_verification_email(self):
             user_email_override={self.user: self.email},
         )
 
+    def accept_pending_requests_for_verified_users(self):
+
+        if self.is_verified:
```

Comment:
It is often clearer to escape early rather than nesting

```suggestion
        if not self.is_verified:
            raise RuntimeError("Refusing to accept verifications for unverified user")
            
        permission_request_classes = ...
```

Reply:
Done